app/bt/nativeModule.ts

```diff
@@ -72,12 +73,43 @@ export const getCurrentENPermissionsStatus = async (
 };
 
 // Key Submission Module
+const keySubmissionModule = NativeModules.KeySubmissionModule;
+
 export const submitDiagnosisKeys = async (
   cb: (errorMessage: string, successMessage: string) => void,
 ): Promise<void> => {
   keySubmissionModule.postDiagnosisKeys(cb);
 };
 
+// Exposure Key Module
+const exposureKeyModule = NativeModules.ExposureKeyModule;
+
+export const getExposureKeys = async (): Promise<ExposureKey[]> => {
+  const keys: RawExposureKey[] = await exposureKeyModule.fetchExposureKeys();
+  return keys.map(toExposureKey);
+};
+
+interface RawExposureKey {
+  key: null | string;
```

Comment:
@mattThousand , noticing that we are getting keys that look like:
```
{"key": null, "rollingPeriod": 144, "rollingStartNumber": 2657520, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2657376, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2657232, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2657088, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656944, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656800, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656656, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656512, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656368, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656224, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2656080, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2655936, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2655792, "transmissionRisk": 0}, {"key": null, "rollingPeriod": 144, "rollingStartNumber": 2655648, "transmissionRisk": 0}
```

from the native layer.

Do you know why the key field is coming in a `null` ?